android/src/main/java/com/firebasePushNotification/FirebasePushNotification.java

```diff
@@ -68,6 +71,24 @@ public void checkPermissionForPushNotification(Callback callback) {
         callback.invoke(null,NotificationManagerCompat.from(getCurrentActivity()).areNotificationsEnabled());
     }
 
+
+    @ReactMethod
+    public void getToken(final Callback callback) {
+        FirebaseInstanceId.getInstance().getInstanceId()
+                .addOnCompleteListener(new OnCompleteListener<InstanceIdResult>() {
```

Comment:
This class does not implement onNewToken case. It should be handled. See [this](https://firebase.google.com/docs/cloud-messaging/android/client#monitor-token-generation) for more details. The same applies to iOS as well.